.gitignore

```diff
@@ -1,4 +1,3 @@
 .DS_Store
 .vscode
-target/
-Cargo.lock
```

Comment:
Should we keep `Cargo.lock` for library?

https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries